temporalio/worker/_workflow.py

```diff
@@ -698,5 +720,39 @@ def attempt_deadlock_interruption(self) -> None:
                 )
 
 
+class _CommandAwarePayloadCodec(temporalio.converter.PayloadCodec):
+    """A payload codec that sets serialization context for the associated command.
+
+    This codec responds to the :py:data:`temporalio.bridge._visitor.current_command_seq` context
+    variable set by the payload visitor.
+    """
+
+    def __init__(
+        self,
+        instance: WorkflowInstance,
+        context_free_payload_codec: temporalio.converter.PayloadCodec,
```

Comment:
Mentioned above, but might as well pass _also_ in the workflow-context codec at this time, you already have it